bpf/init.sh

```diff
@@ -642,8 +645,11 @@ fi
 
 if [ "$XDP_DEV" != "<nil>" ]; then
 	CIDR_MAP="cilium_cidr_v*"
-	COPTS=""
-	xdp_load $XDP_DEV $XDP_MODE "$COPTS" bpf_prefilter.c bpf_prefilter.o from-netdev $CIDR_MAP
+	COPTS="-DSECLABEL=${ID_WORLD} -DCALLS_MAP=cilium_calls_xdp"
+	if [ "$NODE_PORT" = "true" ]; then
+		COPTS="${COPTS} -DLB_L3 -DLB_L4 -DDISABLE_LOOPBACK_LB"
```

Comment:
Maybe a follow-up cleanup but I wonder if we should move towards having a dedicated `xdp_config.h` header, ideally with as much code sharing with `netdev_config.h` as possible.

Reply:
Yeah, agree, any special casing for XDP should be avoided so we can have better integration with the rest of the code.